deploy/olm-catalog/3scale-operator/3scale-operator.v0.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,296 @@
+          version: v1
+        - kind: Service
+          version: v1
+        - kind: Route
+          version: route.openshift.io/v1
+        - kind: ImageStream
+          version: image.openshift.io/v1
+      specDescriptors:
+        - displayName: Wildcard Domain
+          description: Wildcard domain as configured in the API Manager object
+          path: wildcardDomain
+          x-descriptors:
+            - "urn:alm:descriptor:com.tectonic.ui:label"
+      statusDescriptors:
+        - displayName: Deployments
+          description: API Manager Deployment Configs
+          path: deployments
+          x-descriptors:
+            - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
+    - kind: API
+      name: apis.capabilities.3scale.net
+      version: v1alpha1
+      description: API
+      displayName: API
+    - kind: Binding
+      name: bindings.capabilities.3scale.net
+      version: v1alpha1
+      description: Binding
+      displayName: Binding
+    - kind: Limit
+      name: limits.capabilities.3scale.net
+      version: v1alpha1
+      description: Limit
+      displayName: Limit
+    - kind: MappingRule
+      name: mappingrules.capabilities.3scale.net
+      version: v1alpha1
+      description: MappingRule
+      displayName: MappingRule
+    - kind: Metric
+      name: metrics.capabilities.3scale.net
+      version: v1alpha1
+      description: Metric
+      displayName: Metric
+    - kind: Plan
+      name: plans.capabilities.3scale.net
+      version: v1alpha1
+      description: Plan
+      displayName: Plan
+    - kind: Tenant
+      name: tenants.capabilities.3scale.net
+      version: v1alpha1
+      description: Tenant
+      displayName: Tenant
+  description: |
+    The 3scale Operator creates and maintains the Red Hat 3scale API Management on [OpenShift](https://www.openshift.com/) in various deployment configurations.
+
+    [3scale API Management](https://www.redhat.com/en/technologies/jboss-middleware/3scale) makes it easy to manage your APIs.
+    Share, secure, distribute, control, and monetize your APIs on an infrastructure platform built for performance, customer control, and future growth.
+
+    ### Supported Features
+    * **Installer** A way to install a 3scale API Management solution, providing configurability options at the time of installation
+    * **Upgrade** Upgrade from previously installed 3scale API Management solution
+    * **Reconcilliation** Tunable CRD parameters after 3scale API Management solution is installed
+    * **Capabilities** Ability to define 3scale API definitions and set them into a 3scale API Management solution
+
+    ### Documentation
+    [3scale api management](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management)
+    [Deploying 3scale using the operator](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.7/html/installing_3scale/onpremises-installation#deploying-threescale-using-the-operator)
```

Comment:
Documentation team changed the path to this. We should take advantage of this PR and fix it. The new correct link that we should set is:

https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.7/html/installing_3scale/install-threescale-on-openshift-guide#deploying-threescale-using-the-operator